cli/cmd/generate.go

```diff
@@ -63,21 +63,42 @@ func runGenerate(cmd *cobra.Command, args []string) error {
 	}
 }
 
-func genSource(cmd *cobra.Command, path string, pm *plugins.PluginManager, registry specs.Registry, outputFile string) error {
-	if registry == specs.RegistryGithub && !strings.Contains(path, "/") {
-		path = "cloudquery/" + path
-	}
-	version := "latest"
-	if strings.Contains(path, "@") {
-		version = strings.Split(path, "@")[1]
-	}
-
-	sourceSpec := specs.Source{
+func getSourceSpec(path string, registry specs.Registry) specs.Source {
+	if registry == specs.RegistryGithub {
+		if !strings.Contains(path, "/") {
+			path = "cloudquery/" + path
+		}
+
+		nameParts := strings.Split(path, "/")
+		versionParts := strings.Split(nameParts[1], "@")
+
+		org := nameParts[0]
+		name := versionParts[0]
+		version := "latest"
+		if len(versionParts) > 1 {
+			version = versionParts[1]
+			if !strings.HasPrefix(version, "v") {
+				version = "v" + version
+			}
```

Comment:
hmm, I'm wondering whether this is wise--should we maybe rather let it fail if you don't set the `v`? Otherwise maybe a plugin producer can release without the `v` and then you can't use that tag

Reply:
Yeah good point. Lets leave it out for now, and we can always add it later